package.json

```diff
@@ -38,7 +39,7 @@
   },
   "peerDependencies": {
     "less": "^2.3.1",
-    "webpack": "^2.2.0"
+    "webpack": "^2.0.0 || >= 3.0.0-rc.0 || ^3.0.0"
```

Comment:
Pull the RC here, we don't want to officially support pre-release builds now that final is out on npm

`"webpack": "^2.0.0 || ^3.0.0"`